cdc/owner/changefeed.go

```diff
@@ -604,6 +601,7 @@ LOOP2:
 		return err
 	}
 	if c.redoMetaMgr.Enabled() {
+		c.resolvedTs = c.redoMetaMgr.GetFlushedMeta().ResolvedTs
```

Comment:
if it is  a new changefeed , what is the value of resolved ts (meta) stored in  s3?

Reply:
The meta file is created when the redoMetaManager is initialized,  ref: https://github.com/pingcap/tiflow/blob/master/cdc/redo/meta_manager.go#L206-L208.
So, if it is a new changefeed, the value of meta will be equal to startTs